rails/config/initializers/new_framework_defaults.rb

```diff
@@ -15,6 +15,3 @@
 # Make Ruby 2.4 preserve the timezone of the receiver when calling `to_time`.
 # Previous versions had false.
 ActiveSupport.to_time_preserves_timezone = false
-
-# Do not halt callback chains when a callback returns false. Previous versions had true.
-ActiveSupport.halt_callback_chains_on_return_false = true
```

Comment:
This function was added for passivity and then removed. This file was autogenerated during the rails 5 upgrade. 

https://github.com/rails/rails/issues/32653